0016/endpoint-time-series-event.avsc

```diff
@@ -0,0 +1,102 @@
+{
+    "namespace":"org.kaaproject.ipc.collector2ts.gen.v1",
+    "name":"TimeSeriesEvent",
+    "type":"record",
+    "fields":[
+        {
+            "name":"correlationId",
+            "type":"string",
+            "doc":"Message ID primarily used to track message processing across services"
+        },
+        {
+            "name":"timestamp",
+            "type":"long",
+            "doc":"Message creation UNIX timestamp in milliseconds"
+        },
+        {
+            "name":"timeout",
+            "type":"long",
+            "default":0,
+            "doc":"Amount of milliseconds since the timestamp until the message expires. Value of 0 is reserved to indicate no expiration."
+        },
+        {
+            "name":"appVersionName",
+            "type":"string",
+            "doc":"Application version name the data is sent for"
+        },
+        {
+            "name":"endpointId",
+            "type":"string",
+            "doc":"Identifier of the endpoint that data points belong to"
+        },
+        {
+            "name":"timeSeriesName",
+            "type":"string",
+            "doc":"Name of the time series that data points belong to"
+        },
+        {
+            "name":"dataPoints",
+            "doc":"Array of endpoint time series data points",
+            "type":{
+                "type":"array",
+                "items":{
+                    "namespace":"org.kaaproject.ipc.collector2ts.gen.v1",
+                    "name":"DataPoint",
+                    "type":"record",
+                    "fields":[
+                        {
+                            "name":"timestamp",
+                            "type":"long",
+                            "doc":"Data point UNIX timestamp in milliseconds"
+                        },
+                        {
+                            "name":"value",
+                            "type":[
+                                "boolean",
+                                "int",
+                                "long",
+                                "float",
+                                "double",
+                                "string",
+                                "bytes"
```

Comment:
Add `null`. It is a valid value

Reply:
IMHO, `null` conflicts with the definition of the data point, which says that value is mandatory.

Reply:
`null` is a useful value to identify an absent value. (We already support displaying gaps in the chart for this case.)

Reply:
I'm still not certain about this, but let's add `null`, if you insist.